interfaces/builtin/content.go

```diff
@@ -78,6 +78,18 @@ func (iface *contentInterface) BeforePrepareSlot(slot *snap.SlotInfo) error {
 		slot.Attrs["content"] = slot.Name
 	}
 
+	// Error if "read" or "write" are present alongside "source".
+	var unused map[string]interface{}
```

Comment:
Note that this will only catch `source` if it's a map. But that's probably fine?

Reply:
Aha, good catch. I will change this to interface{} and ensure it is tested.